plugins/Atric_IRWakeupUSB/__init__.py

```diff
@@ -0,0 +1,1511 @@
+# -*- coding: utf-8 -*-
+#
+# This file is a plugin for EventGhost.
+# Copyright © EventGhost Project <http://www.eventghost.net/>
+#
+# EventGhost is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free
+# Software Foundation, either version 2 of the License, or (at your option)
+# any later version.
+#
+# EventGhost is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with EventGhost. If not, see <http://www.gnu.org/licenses/>.
+
+
+import eg
+
+
+eg.RegisterPlugin(
+    name="Atric IR-WakeupUSB",
+    guid="{29CC33EC-2304-4879-9B5B-D98ECCADC5B6}",
+    author=("GruberMarkus"),
+    version="0.1",
+    kind="remote",
+    createMacrosOnAdd=True,
```

Comment:
As you mention to contact you through Github, you can add the link here
`url="https://github.com/GruberMarkus/EventGhost",`
and it will appear in the plugin config dialog.

But maybe a new thread in the forums plugin section would be a good idea.

Reply:
Thanks for the hint, I already added the link to the source code. I will create a thread in the EventGhost forum as soon as plugin is publicy available.